agent/engine/docker_task_engine.go

```diff
@@ -443,6 +443,8 @@ func (engine *DockerTaskEngine) createContainer(task *api.Task, container *api.C
 	}
 
 	hostConfig, hcerr := task.DockerHostConfig(container, containerMap)
+	hostConfig.NetworkMode = "host"
```

Comment:
Honestly, I dont have much understanding of Go to be able to verify if that is the case. I just wanted to try out the changes as made here - https://github.com/chaos-generator/amazon-ecs-agent/commit/e139aaa61c410704b71915be3051154466b1bec1 on the latest amazon-ecs-agent.